action.yml

```diff
@@ -0,0 +1,31 @@
+name: Remote tflint Config
+description: Copies a tflint configuration file from a remote repo for local use
+author: ryanwholey
+inputs:
+  source-repo: 
+    required: true
+    description: The repository to copy a tflint configuration from
+  source-path:
+    description: The directory in the remote repo where the configuration exists
+    default: .
+  source-filename:
+    description: The source filename
+    default: .tflint.hcl
+  source-ref:
+    description: Ref or branch of the remote repository to target
+    default: main
+  destination-path:
+    description: Directory to write the config file
+    default: ${{ runner.temp }}
+  destination-filename:
+    description: Name of the configuration file to be written
+    default: .tflint.hcl
+  token:
+    required: false
+    description: Github access token, required for private repositories
```

Comment:
Presumably this would have to be a PAT to load a private org repo? If so, that's worth clarifying somewhere.

Reply:
Updated but only slightly, lmk what you think